azext_iot/digitaltwins/commands_jobs.py

```diff
@@ -4,14 +4,16 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
+from typing import Optional
 from azext_iot.digitaltwins.providers.import_job import ImportJobProvider
 from azext_iot.digitaltwins.providers.deletion_job import DeletionJobProvider
 
 
 def create_import_job(
     cmd, name_or_hostname: str, data_file_name: str, input_blob_container_name: str,
-    input_storage_account_name: str, output_file_name: str = None, output_blob_container_name: str = None,
-    output_storage_account_name: str = None, job_id: str = None, resource_group_name: str = None
+    input_storage_account_name: str, output_file_name: Optional[str] = None,
+    output_blob_container_name: Optional[str] = None, output_storage_account_name: Optional[str] = None,
+    job_id: Optional[str] = None, resource_group_name: Optional[str] = None
```

Comment:
just a personal nit - the formatting here makes it hard to tell (in diffs) which params were added / removed / renamed

Reply:
alright - will expand so it will be easier